d2bs/kolbot/libs/common/Pather.js

```diff
@@ -617,6 +617,94 @@ ModeLoop:
 					/* i < areas.length - 1 is for crossing multiple areas.
 						In that case we must use the exit before the last area.
 					*/
+					if (targetArea == 65) {
+						unit = getUnit(2, 74);
+						if (unit) {
+							switch (unit.mode) {
+								case 0:
+								sendPacket(1, 0x13, 4, 0x02, 4, unit.gid);
+								delay(500);
+								break;
+								case 2:
+								break;
+							}
+							sendPacket(1, 0x13, 4, 0x02, 4, unit.gid);
+							return true;
+						}
+						if (!unit) {
+							throw new Error("moveToExit: Unit not found.");
+						}
+					} else if (me.area == 92 && targetArea == 93) {
+						this.moveToPreset(92, 2, 366);
+						lever = getUnit(2, 367);
```

Comment:
There is no check on the "lever" object to see if it is valid before packet is sent.